.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+registries:
+  rubygems-jfrog:
+    type: rubygems-server
+    url: https://clio.jfrog.io/clio/api/gems/product-gem-prod/
+    username: ${{secrets.ARTIFACTORY_USERNAME}}
+    password: ${{secrets.ARTIFACTORY_API_KEY}}
+
+updates:
+  - package-ecosystem: "bundler"
+    directory: "/"
+    schedule:
+      interval: monthly
+    open-pull-requests-limit: 0
```

Comment:
👍  Achieving the stated goal.
https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#open-pull-requests-limit